octavia-cli/octavia_cli/apply/resources.py

```diff
@@ -597,6 +601,16 @@ def _deserialize_raw_configuration(self):
         configuration["namespace_definition"] = NamespaceDefinitionType(configuration["namespace_definition"])
         if "schedule" in configuration:
             configuration["schedule"] = ConnectionSchedule(**configuration["schedule"])
+        if "schedule_type" in configuration:
+            # Note: these must both be supplied.
+            configuration["schedule_type"] = ConnectionScheduleType(configuration["schedule_type"])
+            if "basic_schedule" in configuration["schedule_data"]:
+                basic_schedule = ConnectionScheduleDataBasicSchedule(**configuration["schedule_data"]["basic_schedule"])
+                configuration["schedule_data"]["basic_schedule"] = basic_schedule
+            if "cron" in configuration["schedule_data"]:
+                cron = ConnectionScheduleDataCron(**configuration["schedule_data"]["cron"])
+                configuration["schedule_data"]["cron"] = cron
+            configuration["schedule_data"] = ConnectionScheduleData(**configuration["schedule_data"])
```

Comment:
is there a reason why the manual type isn't here?

Reply:
When the type is manual, then `schedule_data` is null, so we only need to handle the schedule_type (in line 606).